grakn-graph/src/main/java/ai/grakn/graph/internal/computer/GraknComputerImpl.java

```diff
@@ -118,4 +136,22 @@ protected GraphComputer getGraphComputer() {
         return graph.compute(this.graphComputerClass);
     }
 
+    private void applyFilters(Set<LabelId> types, boolean includesShortcut) {
+        if (types == null || types.isEmpty()) return;
```

Comment:
For now, if subgraph is not specified, shortcut edges must be included. Other wisethere is no useful edges in the graph for message passing.